lib/prop/leaky_bucket_strategy.rb

```diff
@@ -17,7 +17,8 @@ def counter(cache_key, options)
 
       # WARNING: race condition
       # this increment is not atomic, so it might miss counts when used frequently
-      def increment(cache_key, options)
+      def change(cache_key, options)
+        raise ArgumentError, "decrement is not supported for LeakyBucketStrategy" if options.key?(:decrement)
```

Comment:
... I'm not sure how it works and if negative numbers could be bad ... so maybe good enough this way ...

`# please open a PR if you know how to support this` ... or ping @shanitang